src/clj/qbits/jet/client/http.clj

```diff
@@ -323,7 +338,8 @@
                               (fold-chunks+decode-xform as fold-chunked-response-buffer-size)
                               (decode-chunk-xform as)))
         trailers-ch (async/promise-chan)
-        ^Request request (.newRequest client ^String url)]
+        ^Request request (.newRequest client ^String url)
+        trailers-supported? (and trailers-fn (instance? HttpRequest request))]
```

Comment:
What other types can `request` be?

Reply:
The instance check is for safety while casting it. 
The `Request` interface has one other implementation: `WebSocketUpgradeRequest`. 
I added the check because I ran into an unsafe cast error on the `Response`.